checkout/access-checkout-android/src/main/java/com/worldpay/access/checkout/api/HttpClient.kt

```diff
@@ -164,6 +164,10 @@ internal class HttpClient(private val urlFactory: URLFactory = URLFactoryImpl(),
         private const val POST_METHOD = "POST"
         private const val GET_METHOD = "GET"
         private const val LOCATION = "Location"
+
+        private const val CONNECT_TIMEOUT = 30000
+        private const val READ_TIMEOUT = 30000
```

Comment:
Would be nice to externalise these constants into resources

Reply:
👍